services/EventTokenService.js

```diff
@@ -0,0 +1,69 @@
+const jwt = require("jsonwebtoken");
+const uuid4 = require("uuid4");
+
+// A service class for Token generation and management
+logger.info(config.get("Event100ms.APP_ACCESS_KEY"));
```

Comment:
Why log the config of APP_ACCESS_KEY

Reply:
+1, don't log or print the secret tokens.

Reply:
Sorry, fixed it now. I would have used it while debugging. Missed to remove it.